utils/apparmor/Makefile

```diff
@@ -0,0 +1,212 @@
+# SPDX-License-Identifier: GPL-1.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apparmor
+PKG_VERSION:=3.0.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.com/apparmor/apparmor.git
+PKG_SOURCE_VERSION:=0325ba06da6eeb5acf3e568063a08136fd0913e0
+PKG_MIRROR_HASH:=303ceca041ad8023fa44cdda366448d60b6299790266834b4078d30b70ad27f9
+
+PKG_LICENSE:=GPL-1.0-only
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_BUILD_DEPENDS:=python3
+BUILDONLY:=1
```

Comment:
It would, - but without BUILDONLY, make install will be called and as source package contains sub-packages, it doesn't work as expected, so it's there for a reason..

Reply:
This is completely wrong. BUILDONLY cannot be placed here.